From f37947a7e083532676a9f2ed079dff6bdc19a8e9 Mon Sep 17 00:00:00 2001
From: Sabari Kumar Murugesan <smurugesan@vmware.com>
Date: Tue, 15 Sep 2015 14:22:11 -0700
Subject: [PATCH] Fix swift store tests for latest swiftclient

The latest swiftclient (2.6.0) breaks some of the swift store
tests as a mock function's parameters got changed.

Change-Id: I36512fbe642f4f12cf1382fdf0e37eccbf1acba4
---
 glance_store/tests/unit/test_swift_store.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/glance_store/tests/unit/test_swift_store.py b/glance_store/tests/unit/test_swift_store.py
index f738cf9..3fe4699 100644
--- a/glance_store/tests/unit/test_swift_store.py
+++ b/glance_store/tests/unit/test_swift_store.py
@@ -92,7 +92,7 @@ def fake_head_container(url, token, container, **kwargs):
     def fake_put_container(url, token, container, **kwargs):
         fixture_containers.append(container)
 
-    def fake_post_container(url, token, container, headers, http_conn=None):
+    def fake_post_container(url, token, container, headers, **kwargs):
         for key, value in six.iteritems(headers):
             fixture_container_headers[key] = value
 
